### ISSUE TITLE
Satellite Radio Control Preset

### ISSUE THREAD

**Issue by abraxasneo** [f4exb/sdrangel#1674]
I have version 17.13.0 and am working with RSPDuo sdr. 
The problem is that the frequency presets for the selected satellites are not displayed.
 I am trying with different satellites but to no avail. 
![image](https://user-images.githubusercontent.com/28750844/235077845-ebd5dd70-1216-4b92-aa54-fe874b87011c.png)

**Comment by abraxasneo** [comment]
There are no presets there when this dialog is opened. That is why my question. Maybe it's just my simple mistake, but I've run out of ideas
![image](https://user-images.githubusercontent.com/28750844/235080718-c266cc7d-7dae-406c-be1d-508526c0df7d.png)

**Comment by srcejon** [comment]
Yes, this displays Device Set presets - which you need to create yourself.

Close that dialog and setup SDRangel how you need it for that particular satellite - then save a Device Set preset using the Star button in the Device window. Then that preset will appear in the combo box you've highlighed.

**Comment by abraxasneo** [comment]
OK :) Thank you!
 I naively thought that the program automatically matches frequencies and demodulators to the appropriate satellite modes....
